redis_enterprise/manifest.json

```
@@ -57,8 +57,10 @@
       "redis-enterprise-overview": "assets/dashboards/redis_enterprise_overview.json",
       "redis-enterprise-database": "assets/dashboards/redis_enterprise_database.json",
       "redis-enterprise-node": "assets/dashboards/redis_enterprise_node.json",
-      "redis-enterprise-replication": "assets/dashboards/redis_enterprise_replication.json",
-      "redis-enterprise-proxy": "assets/dashboards/redis_enterprise_proxy.json"
+      "redis-enterprise-active-active": "assets/dashboards/redis_enterprise_active-active.json",
+      "redis-enterprise-proxy": "assets/dashboards/redis_enterprise_proxy.json",
+      "redis-enterprise-proy-threads": "assets/dashboards/redis_enterprise_proxy-threads.json",
```

Should this be `proxy-threads?
```suggestion
      "redis-enterprise-proxy-threads": "assets/dashboards/redis_enterprise_proxy-threads.json",
```

---

good catch